x/wasm/keeper/genesis_test.go

```diff
@@ -642,6 +653,14 @@ func setupKeeper(t *testing.T) (*Keeper, sdk.Context, []sdk.StoreKey) {
 	}, false, log.NewNopLogger())
 
 	encodingConfig := MakeEncodingConfig(t)
+	// register an example extension. must be protobuf
+	encodingConfig.InterfaceRegistry.RegisterImplementations(
+		(*types.ContractInfoExtension)(nil),
+		&govtypes.Proposal{},
+	)
+	// also registering gov interfaces for nested Any type
+	govtypes.RegisterInterfaces(encodingConfig.InterfaceRegistry)
```

Comment:
Interesting example - and that this may be a general interface not just one type.

Reply:
yes, there can be multiple implementations registered.